shared/js/ui/base/localize.es6.js

```diff
@@ -0,0 +1,36 @@
+import i18next from 'i18next'
+import ICU from 'i18next-icu'
+
+const supportedLocales = require('../../../data/constants.js').supportedLocales
```

Comment:
Good point, I updated `supportedLocales` to be imported since that's a module, the languages I left along as they are JSON and should be refactored anyway.